frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { testName, checkErrors } from '../../support';
+import {
+  SnapshotDetails,
+  snapshotClassDropdown,
+  dropdownFirstItem,
+  pvcDropdown,
+} from '../../views/snapshot';
+import { listPage } from '../../views/list-page';
+import { PVC, testerDeployment, SnapshotClass } from '../../mocks/snapshot';
+import { detailsPage } from '../../views/details-page';
+import { submitButton } from '../../views/form';
+
+const snapshotName = `${PVC.metadata.name}-snapshot`;
+
+/**
+ * This suite works only with AWS platform.
+ */
+
+describe('Snapshot Tests', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+    cy.exec(`echo '${JSON.stringify(PVC)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(testerDeployment)}' | oc apply -n ${testName} -f -`);
+    cy.exec(`echo '${JSON.stringify(SnapshotClass)}' | oc apply -f -`);
+    // Let the PVC get bound
+    cy.wait(10 * 1000);
```

Comment:
`waits` are frowned upon in general, is there a GUI page or component we can test for which will be in the DOM after PVC 'gets bound'?

Reply:
So to track the bound/failed status we would either have to poll the command line or go to the PVC page and wait on this resource.

Reply:
"go to the PVC page and wait on this resource" sounds more Cypress-y :-)